composer.json

```diff
@@ -22,16 +22,16 @@
             "name": "David Dan", "email": "davidwdan@gmail.com", "role": "Developer"
         }
     ],
-    "require": {
-        "react/http": "^0.4.1",
-        "react/http-client": "^0.4.8",
-        "voryx/event-loop": "^0.2.0",
-        "reactivex/rxphp": "^1.0",
-        "guzzlehttp/psr7": "^1.2"
-    },
     "autoload": {
         "psr-4": {
-            "Rx\\Websocket\\": "src/Websocket/"
+            "Rx\\Websocket\\": "src/"
         }
+    },
+    "require": {
+        "react/http": "^0.4.1",
+        "react/http-client": "^0.4.10",
+        "voryx/event-loop": "^0.2.1",
```

Comment:
I don't know what you guys are talking about.  Everything looks good to me :smile: